File: KODY/Podstawy/Hello.cpp
#include <iostream>

using namespace std;

#define ROK 2017
int parzysta(int b){
    int i;
    for (i=0; i<=b; i+=2)
        cout << i << " ";
    return i / 2;
    }


int main(int argc, char **argv) {   
    
    //~const int rok = 2017;
    //string imie;
    const int rokcpp = 1983;
    char imie[20]; 
    int a = 0;
    
    cout << "Podaj imie: ";
    cin >> imie;
    cout << "Witaj " << imie << endl;
    cout << "Ile masz lat? ";
    cin >> a ;
    cout << "Urodziłeś się w: " << ROK - a << endl;
    
    if ((ROK-a)>rokcpp)
        cout <<"Jestem starszy~C++"<< endl;
	else if ((ROK-a)<rokcpp)
        cout << "Jestem młodszy~C++" << endl;
    else
        cout <<"Jesteśmy w tym samym wieku" ;
    
    cout << endl;
    
    
    int b;
    cout << "Podaj liczbę: ";
    cin >> b;
    cout << "Ilość parzystych"<< parzysta(b);
    return 0;
}

File: KODY/Podstawy/Hello.py
#! /usr/bin/env python
# -*- coding: utf-8 -*-

ROK = 2017
ROKPY = 1991

def parzysta(b):
    ile = list(range(0, b+1, 2))
    return ile

def main(args):
    imie = input("Jak masz na imię? ")
    wiek = int(input("Ile masz lat? "))
    rokur = ROK - wiek
    print ("Czołem! ", imie)
    print ("No to jesteś z: ", ROK-wiek) 
    if ROKPY < rokur:
        print("Jestem starszy~PY")
    elif ROKPY > rokur:
        print ("Jestem młodszy~PY")
    elif ROKPY == rokur:
        print ("Rówieśnik~PY")
    
    b = int(input("podaj liczbę naturalną: "))
    print('Ilość parzystych:', parzysta(b))
    
    return 0
    
if __name__ == '__main__':
    import sys
    sys.exit(main(sys.argv))
